pipeline/eval/eval.py

```diff
@@ -136,6 +145,16 @@ def main(args_list: Optional[list[str]] = None) -> None:
     parser.add_argument(
         "--model_variant", type=str, help="The model variant to use, (gpu, cpu, quantized)"
     )
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        help="The name of the dataset (e.g. 'flores-aug-inline-noise_devtest')",
+    )
+
+    # Add Weight & Biases CLI args when module is loaded
+    if WANDB_AVAILABLE:
+        add_wandb_arguments(parser)
```

Comment:
Why are you extending the command line args of another script? Keep in mind that W&B publishing will be used in a bunch of other places, for example for dataset analysis. I understand that the idea is to reuse the code but this is way too hacky and unnecessary. All you need for the publisher class is to accept optional parameters and read the environment variables by default. Also, this makes the `cli/taskcluster.py` less readable.

Reply:
@vrigal I don't think this was resolved. The idea is not to use `parser` of `eval.py` but call  `get_wandb_publisher` with the arguments either manually loaded from the environment variables or maybe using another method from `tracking`.